packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -321,6 +321,7 @@ export interface IDetailsListProps extends IBaseProps<IDetailsList>, IWithViewpo
    * Whether to animate updates
    */
   enableUpdateAnimations?: boolean;
+  ariaLabelForToggleAllGroupsButton?: string;
```

Comment:
Instead of adding this prop better to expose another one called `detailsHeaderProps` of type `IDetailsHeaderProps` that already is having the prop you are adding here and many more others that a user might find useful to override and later just spread it in the `onRenderDetailsHeader` call. @KevinTCoughlin FYI